src/core/attached_body.cpp

```diff
@@ -11,14 +11,14 @@ DEFINE_TEMPLATE_ATTACHED_BODY(double);
 template <typename S>
 AttachedBodyTpl<S>::AttachedBodyTpl(const std::string &name, const FCLObjectPtr &object,
                                     const ArticulatedModelPtr &attached_articulation,
-                                    int attached_link_id, const Isometry3<S> &pose,
+                                    int attached_link_id, const Pose<S> &pose,
                                     const std::vector<std::string> &touch_links)
     : name_(name),
       object_(object),
       attached_articulation_(attached_articulation),
       pinocchio_model_(attached_articulation->getPinocchioModel()),
       attached_link_id_(attached_link_id),
-      pose_(pose),
+      pose_(pose.toIsometry()),
```

Comment:
Same here just wondering why `pose_` is not in `Pose` since you have built all math operations around it already.

Reply:
See about comment.